src/nvim/api/ui.c

```diff
@@ -160,6 +168,12 @@ void nvim_ui_detach(uint64_t channel_id, Error *err)
     return;
   }
   remote_ui_disconnect(channel_id);
+
+  dict_T *dict = get_vim_var_dict(VV_EVENT);
+  tv_dict_add_nr(dict, S_LEN("channel_id"), (long)channel_id);
```

Comment:
`nvim_list_uis()` uses `chan`, perhaps the same here for consistency.

Reply:
Ok renamed to chan